suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -635,9 +635,22 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
             return;
         }
 
+        const enabledNetworks = selectDeviceEnabledDiscoveryNetworkSymbols(getState());
+
+        let availableCardanoDerivations: ('normal' | 'legacy' | 'ledger')[] = [];
+        if (enabledNetworks.some(network => network.networkType === 'cardano')) {
```

Comment:
TS broken. This does what is intended :slightly_smiling_face: 

```suggestion
        if (enabledNetworks.some(networkSymbol => getNetwork(networkSymbol).networkType === 'cardano')) {
```